package.json

```diff
@@ -78,7 +77,7 @@
   },
   "scripts": {
     "build": "fec build",
-    "test": "jest --verbose",
+    "test": "TZ=UTC jest --verbose --no-cache",
```

Comment:
I know that we require the `--no-cache`option, but instead of adding this flag in the script, can we add it in the CI job that is running this script? The cache is valuable for frontend development.

Reply:
I don't know how to do that are you available to help?

Reply:
It would need to happen somewhere in the operator. Whatever is calling the `test` script. Adding `npm run test -- --no-cache` should do the trick (the extra `--` is required to pass the args down to the nested script)

Reply:
We can merge the changes now, but I would like to make the change in the operator/pr_check to add the no cache argument instead of embedding it into the source repo.

Reply:
I was thinking about ways to do that by adding a new script target, but extra args in the build container would be much cleaner than adding an entry to every package.json